.ci/packaging.groovy

```diff
@@ -118,8 +118,10 @@ pipeline {
           }
           post {
             failure {
-              notifyStatus(subject: "[${env.REPO}@${env.BRANCH_NAME}] package failed.",
-                           body: 'Contact the Productivity team [#observablt-robots] if you need further assistance.')
+              if (!isPR()) {
```

Comment:
```suggestion
              whenTrue(isBranch()) {
```

https://github.com/elastic/apm-pipeline-library/blob/main/vars/isBranch.groovy and https://github.com/elastic/apm-pipeline-library/blob/main/vars/whenTrue.groovy for the declarative pipeline. If conditionals require some tricks with the scripts closure